docs/dynamic-bundling/index.md

```diff
@@ -0,0 +1,41 @@
+# Dynamic Bundling
+
+One way to address the gap in serving primitives is a technique we’ll call “dynamic bundling” here.
+Traditional asset deployments require that there’s a strict 1-to-1 mapping between output files generated at build time and network requests at runtime.
+The output files of the build process are served by a simple file server without any additional transformations beyond compression.
+
+Dynamic bundling adds a runtime component that allows for assembly of network responses from file fragments (“chunks”) generated at build time.
+This makes using a simple file server impossible but offers the following advantages:
+
+* Reduced network requests caused by code splitting.
+  Very fine-grained splitting strategies can be used without the overhead and latency anomalies that come with tens or hundreds of individual requests.
+* Reduced number of output files (and build runtime) when generating many variants.
+  E.g. each use of a feature flag (or experiment) can be isolated into one small chunk.
```

Comment:
nit: is the "small" qualifier for "small chunk" the relevant concept/idea here, or is the important concept that they exist as a separate entity? In other words, can we remove "small" from this sentence and convey the same message with more clarity?

Reply:
```suggestion
  E.g. each use of a feature flag (or experiment) can be isolated into one chunk.
```